css/CSS2/cascade-import/cascade-import-002.xht

```diff
@@ -65,10 +61,6 @@
    f   Rule
       <meta http-equiv="Link">
```

Comment:
This looks more like the comments are associated with the letters after them rather than the letters before them.